examples/libp2p-in-the-browser/1/package.json

```
@@ -23,7 +23,7 @@
     "libp2p-spdy": "~0.12.1",
     "libp2p-webrtc-star": "~0.15.3",
     "libp2p-websocket-star": "~0.8.1",
-    "libp2p-websockets": "~0.12.0",
-    "peer-info": "~0.14.1"
+    "libp2p-websockets": "^0.12.0",
```

semver version match for libp2p-websockets should stay the same.
```suggestion
    "libp2p-websockets": "~0.12.0",
```